src/main/java/io/stargate/sgv2/jsonapi/exception/ErrorCode.java

```diff
@@ -178,7 +178,9 @@ public enum ErrorCode {
   SERVER_QUERY_EXECUTION_FAILURE("Database query execution failed"),
   SERVER_COORDINATOR_FAILURE("Coordinator failed"),
   SERVER_FAILURE("Driver failed"),
-  SERVER_UNHANDLED_ERROR("Server failed");
+  SERVER_UNHANDLED_ERROR("Server failed"),
+
+  INVALID_PARAMETER_VALIDATION_TYPE("Invalid Parameter Validation Type");
```

Comment:
Should this be more specific that this is about embedding providers (both error code and message)? Otherwise without context it is hard to know what this relates to,

Reply:
This should not happen, since it is yaml.file based and not user provided.

Reply:
Hmmh. If that should not happen why would need error code? :)

What I mean is just that even if it's due to configuration problem (bad yaml), error code and message should indicate what is failing.